code/modules/mob/living/carbon/species.dm

```diff
@@ -157,6 +157,13 @@ var/global/list/whitelisted_species = list("Human")
 		myhuman = null
 	..()
 
+/datum/species/proc/gib(var/mob/living/carbon/human/H as mob)
```

Comment:
since this proc doesn't actually handle gibbing (default_gib() does that), maybe a more intuitive name would be before_gib() or pregib() or something.

Reply:
No, it does actually handle gibbing. It's just that each override is expected to perform species specific gibbing. ``default_gib()`` is just the fallback proc for when we don't know what to do or don't have a specific gib case set up for that species. The code in ``proc/gib`` are things that happen every time regardless of species: 
```
	H.death(1)
	H.monkeyizing = 1
	H.canmove = 0
	H.icon = null
	H.invisibility = 101```

Reply:
Yeah but those aren't actually gibbing, those are all things that happen BEFORE gibbing. The throwing organs and creating gibs part is the actual gibbing.